app/core/models.py

```diff
@@ -80,3 +80,28 @@ def get_absolute_url(self):
     def __str__(self):
         """String for representing the Model object."""
         return f'{self.id}'
+
+
+class SearchSortOption(models.Model):
```

Comment:
Recommend inheriting from TimeStampedModel for all of our Django models in order to add some automatic created/modified date stamping capability:  https://django-model-utils.readthedocs.io/en/latest/models.html